src/sparseml/optim/helpers.py

```diff
@@ -401,7 +401,10 @@ def _load_yaml_str_from_file(file_path: Union[str, File]) -> str:
     if file_path.startswith("zoo:"):
         # download from zoo stub
         model = Model(file_path)
-        file_path = model.recipes.default.path
+        recipe = model.recipes
+        if isinstance(recipe, List):
```

Comment:
is the recipe ever not a list? i believe this is what `default` was supposed to handle - selecting the correct recipe out of the list of possible ones. is `default` no longer supported?

Reply:
In the current sparsezoo code `self.recipes` is always a list, see https://github.com/neuralmagic/sparsezoo/blob/main/src/sparsezoo/model/model.py#L153. Some of the unit tests were throwing errors because `model.recipes.default` is no longer valid